pom.xml

```diff
@@ -316,4 +316,81 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>sonatype-oss-release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-source-plugin</artifactId>
+                        <version>${version.maven-source-plugin}</version>
+                        <executions>
+                            <execution>
+                                <id>attach-sources</id>
+                                <goals>
+                                    <goal>jar-no-fork</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>${version.maven-javadoc-plugin}</version>
+                        <configuration>
+                            <taglets>
+                                <taglet>
+                                    <tagletClass>org.codehaus.plexus.javadoc.PlexusComponentTaglet</tagletClass>
+                                    <tagletArtifact>
+                                        <groupId>org.codehaus.plexus</groupId>
+                                        <artifactId>plexus-javadoc</artifactId>
+                                        <version>1.0</version>
+                                    </tagletArtifact>
+                                </taglet>
+                                <taglet>
+                                    <tagletClass>org.codehaus.plexus.javadoc.PlexusConfigurationTaglet</tagletClass>
+                                    <tagletArtifact>
+                                        <groupId>org.codehaus.plexus</groupId>
+                                        <artifactId>plexus-javadoc</artifactId>
+                                        <version>1.0</version>
+                                    </tagletArtifact>
+                                </taglet>
+                                <taglet>
+                                    <tagletClass>org.codehaus.plexus.javadoc.PlexusRequirementTaglet</tagletClass>
+                                    <tagletArtifact>
+                                        <groupId>org.codehaus.plexus</groupId>
+                                        <artifactId>plexus-javadoc</artifactId>
+                                        <version>1.0</version>
```

Comment:
you could also do the properties / version for plexus-javadoc for this particular profile (saves us from having to change it in 3 spots next time)